mqtt/mqtt-bridge/src/settings.rs

```diff
@@ -326,181 +289,53 @@ impl RingBufferSettings {
     }
 }
 
-#[cfg(test)]
-mod tests {
```

Comment:
all these test cases moved to `mqtt-edgehub/src/settings.rs`, not as is, but with similar coverage.